rpcd/playbooks/roles/rpc_maas/tasks/package_install.yml

```diff
@@ -26,6 +26,4 @@
     name: "{{ item }}"
     state: present
     extra_args: "{{ pip_install_options | default('') }}"
-  with_flattened:
-    - maas_pip_packages
-    - maas_pip_dependencies
+  with_items: maas_pip_packages + maas_pip_dependencies
```

Comment:
I think this may be the cause of what we're [`seeing`](http://jenkins.propter.net/job/RPC-AIO/870/consoleFull):

```
15:20:00 changed: [jrpcaioiad-2c] => (item=ipaddr)
15:20:00 failed: [jrpcaioiad-2c] => (item={u'mount_point': u'/srv', u'storage_network': u'br-storage', u'storage_policies': [{u'policy': {u'default': True, u'index': 0, u'name': u'default'}}], u'replication_network': u'br-storage', u'drives': [{u'name': u'swift1.img'}, {u'name': u'swift2.img'}, {u'name': u'swift3.img'}], u'part_power': 8}) => {"failed": true, "item": {"drives": [{"name": "swift1.img"}, {"name": "swift2.img"}, {"name": "swift3.img"}], "mount_point": "/srv", "part_power": 8, "replication_network": "br-storage", "storage_network": "br-storage", "storage_policies": [{"policy": {"default": true, "index": 0, "name": "default"}}]}, "parsed": false}
15:20:00 OpenSSH_6.6.1, OpenSSL 1.0.1f 6 Jan 2014
15:20:00 debug1: Reading configuration data /etc/ssh/ssh_config
15:20:00 debug1: /etc/ssh/ssh_config line 19: Applying options for *
15:20:00 debug1: auto-mux: Trying existing master
15:20:00 debug1: mux_client_request_session: master session id: 2
15:20:00 Traceback (most recent call last):
15:20:00   File "<stdin>", line 1973, in <module>
15:20:00   File "<stdin>", line 300, in main
15:20:00 AttributeError: 'dict' object has no attribute 'startswith'
15:20:00 
15:20:00 ok: [jrpcaioiad-2c] => (item=requests)
```